requirements/requirements.txt

```diff
@@ -1,17 +1,15 @@
-click==8.0.3
-#
-# We are experiencing crashes with grpcio >= 1.39
-grpcio==1.38.1
-Jinja2==3.0.3
+click==8.1.3
+grpcio==1.46.0
+Jinja2==3.1.2
 pluginbase==1.0.1
-protobuf==3.19.4
+protobuf==3.20.1
 psutil==5.9.0
 ruamel.yaml==0.17.21
 ruamel.yaml.clib==0.2.6
-setuptools==59.6.0
+setuptools==44.1.1
```

Comment:
Whoa, I did not spot this. Could you document why setuptools is pinned now to really old version?

Reply:
Ummm, I have no idea, we are not pinning it.

This is a result of updating the requirements, so presumably one of the newer versions of *something* is requiring a restricted version of setuptools.

Reply:
Too bad this pinning tooling is not printing out reasons for the pins. Pip tools can be made to do that.

Reply:
This is likely due to the version of python that was used to freeze the requirements.